.rubocop.yml

```diff
@@ -1,14 +1,18 @@
-Documentation:
+AllCops:
+  Exclude:
+    - bin/**/*
+    - db/**/*
+    # Because this below isn't under my control. toad.
```

Comment:
I looked up toad and still not sure what it means in this context. Sorry I missed this the first time, but will you please add more to explain what `toad` means or leave it out?